internals/cli/cmd_help.go

```diff
@@ -191,6 +191,11 @@ var helpCategories = []helpCategory{{
 	Commands:    []string{"warnings", "okay"},
 }}
 
+// AddHelpCategory appends an existing help category to the Pebble help manual.
+func AddHelpCategory(categ HelpCategory) {
```

Comment:
Naming nit: `categ` seems like an unusual and unnecessary abbreviation. [Wiktionary lists cat](https://en.wiktionary.org/wiki/cat) as an abbrevation for category, but there's no "categ". Should probably just be `category`.